src/pybind/mgr/dashboard/frontend/src/app/ceph/nfs/nfs-form/nfs-form.component.ts

```diff
@@ -265,20 +265,7 @@ export class NfsFormComponent extends CdForm implements OnInit {
     const fsalValue = this.nfsForm.getValue('name');
     const checkAvailability =
       fsalValue === 'RGW'
-        ? this.rgwSiteService.get('realms').pipe(
-            mergeMap((realms: string[]) =>
-              realms.length === 0
-                ? of(true)
-                : this.rgwSiteService.isDefaultRealm().pipe(
-                    mergeMap((isDefaultRealm) => {
-                      if (!isDefaultRealm) {
-                        throw new Error('Selected realm is not the default.');
-                      }
-                      return of(true);
-                    })
-                  )
-            )
-          )
+        ? this.rgwSiteService.get('realms')
```

Comment:
I am not sure if these two api calls are necessary here, unless an error in `realms`/`filesystem` fetching is considered blocking for the NFS export creation.